src/components/select/examples/select-change-options.tsx

```diff
@@ -19,6 +23,12 @@ export class SelectExample {
     public currentOptionGroup = 0;
 
     private optionGroups: Option[][] = [
+        [],
+        [
+            { text: '', value: '', disabled: true },
+            { text: 'Bart Simpson', value: 'bart' },
+            { text: 'Ned Flanders', value: 'ned' },
+        ],
```

Comment:
Why did you move this set of options? Did it not work as expected in the place it was? Or did things not work as expected when the set with "Luke Skywalker" came immediately after the empty set? Or some other reason? 😄

This set and the set with "Homer Simpson" should be next to each other, because it allows you to test what happens if you have "Ned Flanders" selected in one, and then switch to the other. "Ned Flanders" should be selected in that one as well. But I would assume that doesn't work if there's been a set with no "Ned Flanders" in it in-between.

Reply:
Yes, it should work, I moved it because I found another logic there 😀. 
Empty array ->  2 options plus an empty row -> 3 options -> 3 options plus an empty row -> 4 options.

However, if this makes sense only to me, I will put it back 🙂

Reply:
Haha, yeah, they were actually sorted differently on purpose, because I didn't want the number of options to only increase.

But now I've update the example quite a lot, so it doesn't matter anymore. You can now switch from any group to any other group, and you can recreate the component at any state, to check it can load in that state 👍